veros/core/tke.py

```diff
@@ -61,9 +61,12 @@ def set_tke_diffusivities(vs):
         if vs.enable_idemix:
             Rinumber[...] = np.minimum(Rinumber, vs.kappaM * vs.Nsqr[:, :, :, vs.tau]
                                   / np.maximum(1e-12, vs.alpha_c * vs.E_iw[:, :, :, vs.tau]**2))
-        vs.Prandtlnumber[...] = np.maximum(1., np.minimum(10, 6.6 * Rinumber))
-        vs.kappaH[...] = vs.kappaM / vs.Prandtlnumber
+        if vs.enable_Prandtl_tke:
+            vs.Prandtlnumber[...] = np.maximum(1., np.minimum(10, 6.6 * Rinumber))
+        else:
+            vs.Prandtlnumber[...] = vs.Prandtl_tke0
         vs.kappaM[...] = np.maximum(vs.kappaM_min, vs.kappaM)
+        vs.kappaH[...] = vs.kappaM / vs.Prandtlnumber
```

Comment:
This is a bug. `kappaH` should be computed before `kappaM`. I guess having the testing suite is paying off :)

Reply:
Which in fact doesn't make much sense because thermal diffusion depends on viscosity, which is cutoff after `vs.kappaH[...]` computation in the original source code.

Reply:
I guess, it is something to do with IDEMIX+EKE specifics. 
Waving a dead chicken...

Reply:
Good point. We should ask Carsten Eden about this, looks like a bug in PyOM to me.

Reply:
Current PyOM version has the same formulation (first `kappaH`, then `kappaM`), so I would assume it is deliberate.

Reply:
Alright, but can we make it like when IDEMIX+EKE is switched on then we use PyOM formulation otherwise mine?

Reply:
I'd prefer to keep the current formulation and ask Carsten about it. If it is a bug we should wrap this in 

```python
if not vs.pyom_compatibility_mode:
```

to stay compatible during testing. Or would you expect this to have a major impact on your setup?

Reply:
It will certainly affect diffusivity as `vs.kappaM` seems lower than 2e-4 (cut-off value) deep in the ACC sector.
But I think adding `if vs.pyom_compatibility_mode` to _tke.py_ and option `self.pyom_compatibility_mode = True` to _acc_sector.py_ will solve the issue. Should I proceed with that?

Reply:
I sent an email to Carsten about this. Let's leave it as it is in PyOM until we know what's going on. If we deliberately break compatibility we should be absolutely sure it's an issue.